src/Service.Tests/SqlTests/RestApiTests/Insert/InsertApiTestBase.cs

```diff
@@ -263,9 +263,13 @@ public virtual async Task InsertOneWithSqlInjectionTest(string sqlInjection, str
 
         #region Negative Tests
 
+        /// <summary>
+        /// Test to validate that an exception is encountered when the user specifies primary key route or query string for a POST request via REST.
+        /// </summary>
         [TestMethod]
-        public virtual async Task InsertOneWithInvalidQueryStringTest()
+        public virtual async Task InsertOneWithPrimaryKeyOrQueryStringInURLTest()
         {
+            // Validate that a POST request is not allowed to include a query string in the URL.
```

Comment:
since this test is no longer "InsertOneWithInvalidQueryStringTest" can you use a valid query string instead of `?/id/5001`